lib/toolshed/date.ex

```diff
@@ -0,0 +1,47 @@
+defmodule Toolshed.Date do
+  @moduledoc """
+  This module  replicate `date` unix command and its options.
+  """
+
```

Comment:
```suggestion
  @moduledoc false
```

I don't think that the moduledoc is needed since all of the documentation can be on the `main` function. Plus, we want users to call `date` or `Toolshed.date` and not `Toolshed.Date.main`. Turning off module documentation seems like a way of helping this out.